pennylane_qiskit/aer.py

```diff
@@ -38,6 +38,7 @@ class AerDevice(QiskitDevice):
             or iterable that contains unique labels for the subsystems as numbers (i.e., ``[-1, 0, 2]``)
             or strings (``['ancilla', 'q1', 'q2']``).
         backend (str): the desired backend
+        method (str): the desired method
```

Comment:
Should probably elaborate on what "method" is.  Maybe an example value?

Reply:
```suggestion
        method (str): The desired simulation method. A list of supported simulation
            methods can be returned using `qiskit.Aer.available_methods()`, or by referring
            to the `AerSimulator` [documentation](https://qiskit.org/documentation/stubs/qiskit.providers.aer.AerSimulator.html).
```